exporter/zipkinexporter/zipkin.go

```diff
@@ -176,7 +177,11 @@ func zipkinEndpointFromNode(node *commonpb.Node, serviceName string, endpointTyp
 	return zEndpoint
 }
 
-func (ze *zipkinExporter) stop() error {
+func (ze *zipkinExporter) Name() string {
```

Comment:
Remind me why this needed to be added? Exporter interface? But how it was working before without it?

Reply:
Because the factories return the base trace consumers that don't require name. 
https://github.com/open-telemetry/opentelemetry-service/blob/master/exporter/factory.go#L39-L42

In the next pr, when that method definition is changed,  it would fail to compile.